src/js/components/Field/Field.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * @name Field
+ *
+ * @description
+ *  A field group component is a wrapper for grouping the input with label and messages.  See the teamsnap patterns
+ *  library for more information. https://teamsnap-ui-patterns.netlify.com/patterns/components/field-group.html
+ *
+ * @example
+ *  <FieldGroup name='example' label='Check Me' isInline>
+ *    Field Child Data
+ *  </FieldGroup>
+ *
+ */
+
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import { FieldGroup } from "../FieldGroup";
+import { FieldLabel } from "../FieldLabel";
+import { FieldMessage } from "../FieldMessage";
+import { Input } from "../Input";
+import { Status, Size } from "../../types";
+
+const fieldPropTypes = {
+  type: PropTypes.oneOf(["toggle", "select", "input"]).isRequired,
+  name: PropTypes.string.isRequired,
+  inputProps: PropTypes.any,
+  label: PropTypes.string,
+  placeholder: PropTypes.string,
+  caption: PropTypes.string,
+  status: Status.PropType,
+  ref: PropTypes.any,
+  otherProps: PropTypes.any,
+  leftIcon: PropTypes.node,
+  rightIcon: PropTypes.node,
+  size: Size.PropType,
+  showStatus: PropTypes.bool,
+  showClear: PropTypes.bool,
+  onClearClicked: PropTypes.func,
+  isDisabled: PropTypes.bool,
+};
+
+type FieldType = React.FunctionComponent<
+  PropTypes.InferProps<typeof fieldPropTypes>
```

Comment:
Is this because it takes the children props as arguments?

<img width="719" alt="Screen Shot 2020-12-07 at 12 38 55 PM" src="https://user-images.githubusercontent.com/9888467/101391085-759f9600-3889-11eb-9d73-106d7bfb6ea2.png">

Reply:
Nope. I forgot to attach them to the function component. :) Fixed!